kloppy/domain/models/common.py

```diff
@@ -220,6 +220,9 @@ def __eq__(self, other):
     def set_position(self, time: Time, position: Optional[PositionType]):
         self.positions.set(time, position)
 
+    def reset_positions(self):
+        self.positions.__init__()
```

Comment:
I don't think directly calling `__init__()` is a good idea. I was confused by it.

Can you add a `reset` method to the `Timecontainer` class instead?

```py
    def reset(self):
        """Clears all items from the container."""
        self.items.clear()
  ```